addons/web_editor/static/src/js/editor/odoo-editor/src/OdooEditor.js

```diff
@@ -3159,6 +3160,20 @@ export class OdooEditor extends EventTarget {
                 fragment.appendChild(p);
             }
         }
+
+        // Split elements containing <br> into seperate elements for each line.
+        const brs = fragment.querySelectorAll('br');
+        for (let br of brs) {
+            const block = closestBlock(br);
+            if (['P', 'H1', 'H2', 'H3', 'H4', 'H5', 'H6', 'PRE'].includes(block.nodeName) &&
```

Comment:
I think you could use `paragraphRelatedElements` here. It also includes `BLOCKQUOTE` but I think that's precisely a good thing.

Reply:
That was the very reason of not using `paragraphRelatedElements`, no one would like if `BLOCKQUOTE` would break into multiple `BLOCKQUOTE`s

Reply:
The same could be said of `pre` though. I don't really see what makes blockquote so different.

@pierre-pde Can we have your input please?

Reply:
@deso-odoo After talking with @pierre-pde, the decision is to neither include `BLOCKQUOTE` _or_ `PRE` here.

```suggestion
            if (['P', 'H1', 'H2', 'H3', 'H4', 'H5', 'H6'].includes(block.nodeName) &&
```